CHANGELOG.md

```diff
@@ -88,6 +88,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Make Span exporter configurable ([#8620](https://github.com/opensearch-project/OpenSearch/issues/8620))
 - Change InternalSignificantTerms to sum shard-level superset counts only in final reduce ([#8735](https://github.com/opensearch-project/OpenSearch/pull/8735))
 - Exclude 'benchmarks' from codecov report ([#8805](https://github.com/opensearch-project/OpenSearch/pull/8805))
+- Add support for tracer to create new TracerContextStorage ([#8831](https://github.com/opensearch-project/OpenSearch/pull/8831))
```

Comment:
@Gaganjuneja we really need to clean up this pull request: the checks are failing, the changelog entry is not up to date, the description of the change is also not right. Could you please spend some time on that?

Reply:
Let me check. Thanks!

Reply:
Gradle check are failing for all the PRs, I need to check on that. Otherwise I have fixed the things.